pkgs/development/python-modules/instructor/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, python3
+, fetchPypi
+, buildPythonPackage
+}:
+
+buildPythonPackage rec {
+  pname = "instructor";
+  version = "0.6.4";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Af52TGnkqY/t0cPkHoRfhFqa/tuOpQXAI/fFfMTcM4Y=";
+  };
+
```

Comment:
If you fetch from github, this will also run the unit tests as part of the build. That's especially helpful with the AI/ML landscape changing so fast.)

Reply:
I looked into enabling tests, but it requires an openai key and network for the majority of the tests. So realistically we wouldn't be able to catch regressions this way anyway because it depends on external systems.